NOSE update.html

```diff
@@ -10,40 +10,14 @@
  style="border: 1px solid ; margin: auto; padding: 0.5em; width: 800px;">
 <h1 style="text-align: center;">NOSE Update<br>
 </h1>
-The 0-15 ratings hack and the base weight for different eras hack both
-require a couple NOSE files to be updated so it can properly display
-the new ratings and weights.<br>
-This update also includes the 30-team NHLPA '93 update.<br>
+The 0-15 ratings hack and the base weight for different eras hack both require <a href="https://forum.nhl94.com/index.php?/topic/20863-nose-newtest-versions/">NOSE version 1.2d or higher</a> to properly display the new ratings and weights. This version also includes support for 32-team NHL 94 ROMs.<br>
```

Comment:
I made a new thread that will always contain the latest NOSE version. Please use this URL: https://forum.nhl94.com/index.php?/topic/26368-nose-latest-version-download/

Reply:
Can remove the line about "This version also includes support for 32-team NHL 94 ROMs."